docs/source/third-party-library-support.rst

```diff
@@ -0,0 +1,46 @@
+Third party library support
+===========================
+
+Support for third party libraries is determined via the following:
+
+- Test suites
+- Project integrations
+- Known limitations
+
+Test suites
+-----------
+
+For each third party library that is supported, the following tasks are performed:
+
+- The test suite is configured to run with Django MongoDB Backend
+
+  - Evaluate test runner configuration
+
+    - Depending on the test runner, updating the settings may require copying
+      ``mongo_apps.py`` and ``mongo_migrations`` to a module that is already
+      included in ``sys.path``.
+
+  - Update django settings
+
+    - Replace the database backend with ``django_mongodb_backend``
+    - Replace contrib apps with MongoDB compatible apps
+    - Replace test suite apps with MongoDB compatible apps
+
+  - Update or disable migrations
+
+    - Use MongoDB compatible migrations if not disabled
+
+- The test suite is run with Django MongoDB Backend
+- The test results are recorded
+- The test suite is updated
+
+  - Replace static primary key references with dynamic references or static ``ObjectId`` references
+
+
+Project integrations
+--------------------
+
+Known limitations
+-----------------
+
+- URL configuration
```

Comment:
This is specific to wagtail, right?
We'll need to create sections for each library we've got running within this test suite.

Reply:
> This is specific to wagtail, right?

Of the 5 targeted libraries, Wagtail may be the only library that will require custom URLs to _run_. But views expecting ``int`` and getting ``ObjectId`` are [found in the tests](https://github.com/carltongibson/django-filter/pull/1703) of most, if not all of the target libraries.

> We'll need to create sections for each library we've got running within this test suite.

Yes! [django-filter is the WIP template](https://django-mongodb-cli.readthedocs.io/en/latest/supported-libraries/django-filter.html) for this.